File: ViniciusMiranda/Models/ToDo.cs
namespace ViniciusMiranda.Models;

public class ToDo
{
    public long Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public string Status { get; set; }
    public DateTime CreatedAt { get; set; }
    public long CategoryId { get; set; }
    public Category Category { get; set; }
}

File: viniciusmiranda/src/interfaces/ToDo.ts
import Category from "./Category";

export default interface ToDo {
  id: number;
  title: string;
  description: string;
  status: string;
  category?: Category
  categoryId: number;
}

export interface ICreateToDo {
    title: string;
    description: string;
    categoryId: number;
  }